examples/sample-dapp/index.html

```diff
@@ -1,233 +1,228 @@
 <!DOCTYPE html>
```

Comment:
can we revert the changes to this file please, it's just noise and we should leave the example dapp with a transfer transaction as this is deployed public facing (technically).